src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Scanner;
+
+
+public class Duke {
+
+    public static final String FILEPATH = "data/savedlist.txt";
+
+    public static void printSeperator() {
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void greet() {
+        printSeperator();
+        System.out.println(" Hello! I'm Duke\n" + " What can I do for you?");
+        printSeperator();
+        System.out.println("");
+    }
+
+
+    private static void farewell() {
+        printSeperator();
+        System.out.println(" Bye. Hope to see you again soon!");
+        printSeperator();
+    }
+
+    private static void parseResponse(String response, List list) {
+
+
+        if (response.equals("list")) {
+            printSeperator();
+            list.listDisplay();
+            printSeperator();
+        } else if (response.length() >= 5 && response.substring(0, 5).equals("mark ")) {
+            printSeperator();
+            list.markTask(response.substring(5));
+            printSeperator();
+        } else if (response.length() >= 7 && response.substring(0, 7).equals("unmark ")) {
+            printSeperator();
+            list.unmarkTask(response.substring(7));
+            printSeperator();
+        } else if (response.length() >= 7 && response.substring(0, 7).equals("delete ")) {
+            printSeperator();
+            list.deleteTask(response.substring(7));
+            printSeperator();
+        }
+        else { // add task
+            printSeperator();
+            list.listAdd(response);
+            printSeperator();
+        }
+    }
```

Comment:
This can go as part of a separate class. Duke has nothing to do with parsing. Parser has something to do with parsing